Missionframework/KPLIB_config_old.sqf

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
Any good reason to keeps this file? We can always check how it was via git history.

Reply:
Of course, we can always have a look at it via the history. Will be deleted. 👍